libs/net/dal/Services/ContentService.cs

```diff
@@ -288,17 +288,118 @@ public override Content AddAndSave(Content entity)
     }
 
     /// <summary>
-    /// Update the specified 'entity' in the database.
+    /// Get the changed properties between current values and database values, display to users .
+    /// Ignores system fields type fields to focus on business-relevant changes.
     /// </summary>
-    /// <param name="entity"></param>
-    /// <returns></returns>
-    /// <exception cref="NoContentException"></exception>
+    /// <param name="currentValues">Current property values from the entity being saved</param>
+    /// <param name="databaseValues">Current property values from the database</param>
+    /// <returns>Dictionary of changed properties with their current and database values</returns>
+    private Dictionary<string, object> GetChangedProperties(Microsoft.EntityFrameworkCore.ChangeTracking.PropertyValues currentValues,
+    Microsoft.EntityFrameworkCore.ChangeTracking.PropertyValues? databaseValues)
+    {
+        // Fields to ignore: system fields 
+        var ignoreFields = new HashSet<string> {
+            "Versions",
```

Comment:
When you need the name of a property you should use `nameof(Content.Versions)` so that it is compile time safe.  Magic strings are pain to maintain.